packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -44,6 +44,10 @@ export interface BaseChoiceProps {
    * Additional classes to be added to the root `div` element.
    */
   className?: string;
+  /**
+   * @hide-prop Internal prop used to determine if a Choice is the child of a another component (like ChoiceList or MonthPicker). Used to hide excessive error messages.
+   */
+  _choiceChild?: boolean;
```

Comment:
Right now the doc snapshots are failing, and I don't think we have a working method of hiding any props from the arg tables. We can do it for web components where we created a custom table component, but I don't know how to hide it from the arg tables in the storybook one. The following will only hide it from the storybook controls but not the doc page:

```js
argTypes: {
  _choiceChild: {
    // Until this pattern has solidified, we're not going to advertize this feature.
    table: {
      disable: true,
    },
  },
}
```